packages/block-library/src/columns/style.scss

```diff
@@ -11,7 +11,6 @@
 }
 
 .wp-block-column {
-	margin-bottom: 1em;
```

Comment:
Do you know when this was added and why?

Reply:
Very probably as part of the effort to normalize block margins. That is, create a minimum baseline margin between all blocks. In this case, I _think_ there's a negative margin on the columns (plural) block, therefore it's applied on the column block instead. I'm not sure. 

A way to test is to insert two columns blocks after each other, with the same content, and watch if it has harmonious margins.

Reply:
The first time this rule was added was one year ago:
https://github.com/WordPress/gutenberg/pull/10541

Then other PR's applies changes/moved things but always kept the rule.

At the time the UI for nesting was very different and I think we did not have .block-editor-block-list__block-edit::before with a negative bottom position that achieves a similar effect. Things were done in the columns block which was the only block with nesting. So I think this was only kept because of historic reasons.

Reply:
Feel free to remove it. If it is necessary in a situation we forgot about it will be the tiniest of regressions that hardly anyone will notice, and we can always add it back. 👍 👍

Reply:
Thank you for the your insights @jasmussen. In that case, I guess I just need a review approving to be able to merge the PR :)